share/messages_to_error_log.txt

```diff
@@ -12900,6 +12909,20 @@ ER_KEYRING_COMPONENT_AWS_READ_ONLY_LIMIT
 ER_KEYRING_COMPONENT_AWS_CANNOT_ENCRYPT_READ_ONLY
   eng "CMK was changed and the keyring must be re-encrypted, but the keyring is read only. Set the read_only config parameter to false for re-encryption and reload the component."
 
+ER_AUTHENTICATION_OPENID_OPTION_REGISTER
+  eng "Option registration failed."
+
+ER_AUTHENTICATION_OPENID_OPTION_DEREGISTER
+  eng "Option deregistration failed."
```

Comment:
did you take from PS 9.2 tree? Please use 9.1 branch and ensure any pxb messages are transferred from our current trunk.